src/D2L.Bmx/UpdateChecker.cs

```diff
@@ -0,0 +1,127 @@
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace D2L.Bmx;
+internal static class UpdateChecker {
+
+	public static async Task CheckForUpdatesAsync( BmxConfig config ) {
+		try {
+			var cachedVersion = GetUpdateCheckCache();
+			var localVersion = Assembly.GetExecutingAssembly().GetName().Version;
+			var latestVersion = new Version( cachedVersion?.VersionName ?? "0.0.0" );
+			if( ShouldFetchLatestVersion( cachedVersion ) ) {
+				Console.Error.WriteLine( "Getting new version from github" );
```

Comment:
This isn't in the screenshot so presumably you added this during dev to test. We should probably remove it right?

A little confusing to have it, makes it sound like we are potentially fetching & installing the new version.

Reply:
It was for debug sorry. Will take this out